package.json

```diff
@@ -55,6 +55,7 @@
     "@types/storybook__react": "^5.2.1",
     "awesome-typescript-loader": "^5.2.1",
     "babel-loader": "^8.2.2",
+    "chromedriver": "^89.0.0",
```

Comment:
Please remove the dependency on chromedriver and rollback the changes to the yarn.lock.

Reply:
Generally we won't accept PRs with any changes to the yarn.lock as it's too hard to review and can be used to sneak in bad dependencies.